invokeai/backend/install/model_install_backend.py

```diff
@@ -166,14 +166,18 @@ def install(self, selections: InstallSelections):
         # add requested models
         for path in selections.install_models:
             logger.info(f'Installing {path} [{job}/{jobs}]')
-            self.heuristic_import(path)
+            try:
+                self.heuristic_import(path)
+            except (ValueError, KeyError) as e:
+                logger.error(str(e))
             job += 1
 
         self.mgr.commit()
 
     def heuristic_import(self,
```

Comment:
I get a crash on app start, as it attempts to read my `$INVOKEAI_ROOT/loras/` folder, which contains 3x `.safetensors` loras:
```py
ERROR:    Traceback (most recent call last):
  File "/home/bat/invokeai/.venv/lib/python3.10/site-packages/starlette/routing.py", line 671, in lifespan
    async with self.lifespan_context(app):
  File "/home/bat/invokeai/.venv/lib/python3.10/site-packages/starlette/routing.py", line 566, in __aenter__
    await self._router.startup()
  File "/home/bat/invokeai/.venv/lib/python3.10/site-packages/starlette/routing.py", line 648, in startup
    await handler()
  File "/home/bat/Documents/Code/InvokeAI/invokeai/app/api_app.py", line 58, in startup_event
    ApiDependencies.initialize(
  File "/home/bat/Documents/Code/InvokeAI/invokeai/app/api/dependencies.py", line 121, in initialize
    model_manager=ModelManagerService(config,logger),
  File "/home/bat/Documents/Code/InvokeAI/invokeai/app/services/model_manager_service.py", line 253, in __init__
    self.mgr = ModelManager(
  File "/home/bat/Documents/Code/InvokeAI/invokeai/backend/model_management/model_manager.py", line 348, in __init__
    self.scan_models_directory()
  File "/home/bat/Documents/Code/InvokeAI/invokeai/backend/model_management/model_manager.py", line 793, in scan_models_directory
    imported_models = self.autoimport()
  File "/home/bat/Documents/Code/InvokeAI/invokeai/backend/model_management/model_manager.py", line 849, in autoimport
    new_models_found.update(installer.heuristic_import(path))
  File "/home/bat/Documents/Code/InvokeAI/invokeai/backend/install/model_install_backend.py", line 196, in heuristic_import
    models_installed.update({str(path):self._install_path(path)})
  File "/home/bat/Documents/Code/InvokeAI/invokeai/backend/install/model_install_backend.py", line 225, in _install_path
    model_name = path.stem if info.format=='checkpoint' else path.name
AttributeError: 'NoneType' object has no attribute 'format'
```

I believe the changes to `heuristic_import()` have caused this crash

Reply:
This is a separate bug that is fixed in PR #3663. However I'll port the fix over to this PR as well.